docs/reportdaily.rst

```diff
@@ -45,6 +45,16 @@ Subcommands
 
 .. HINT: Sort the subcommands alphabetically
 
+init
+~~~~
+
+Is required for the first time use of the programm
+User can enter his data directly or he need to answer some questions provided by this command
```

Comment:
I'd suggest to make it parallel with the other (sub)commands and starts with a verb:

```suggestion
Initializes global user data (required for the first time).
Either user data can be entered directly via options or user will be asked.
```